Version 37: fix htree rename-within-same-dir (b=3417), endianness (b=2447)
Version 36: don't dput dentry after error (b=2350), zero page->private (3119)
Version 35: pass intent to real_lookup after revalidate failure (b=3285)
Version 34: fix ext3 iopen assertion failure (b=2517, b=2399)

 include/linux/lustre_version.h |    1 +
 1 files changed, 1 insertion(+)

Index: lum/include/linux/lustre_version.h
===================================================================
--- lum.orig/include/linux/lustre_version.h	2003-01-30 05:24:37.000000000 -0500
+++ lum/include/linux/lustre_version.h	2004-06-07 17:19:23.000000000 -0400
@@ -0,0 +1 @@
+#define LUSTRE_KERNEL_VERSION 500
